### ISSUE TITLE
Allow lyrics to be spaced out for stage view

### ISSUE THREAD

**Issue by perfectsoundandvideo** [quelea-projection/Quelea#349]
Allow the lyrics to be spaced out more on the stage view because the cords start to overlap

**Comment by berry120** [comment]
Is this still an issue in 2021.0? I feel like we did have this as a problem but it should largely be solved now. If not and you could provide us with an example, that'd be great.

**Comment by perfectsoundandvideo** [comment]
No. The only glitch is that you can not use tabs as spacers and the text has to be aligned left
